examples/dmd/heat_conduction_dmdc.cpp

```diff
@@ -0,0 +1,1028 @@
+    double kappa = 0.5;
+    double closest_rbf_val = 0.9;
+    int rdim = -1;
+    bool offline = false;
+    bool online = false;
+    bool predict = false;
+    bool visualization = true;
+    bool visit = false;
+    int vis_steps = 5;
+    bool adios2 = false;
+    bool save_dofs = false;
+    bool csvFormat = true;
+    const char *basename = "";
+
+    bool pointwiseSnapshots = false;
+    int pwx = 0;
+    int pwy = 0;
+    int pwz = 0;
+
+    int precision = 8;
+    cout.precision(precision);
+
+    OptionsParser args(argc, argv);
+    args.AddOption(&mesh_file, "-m", "--mesh",
+                   "Mesh file to use.");
+    args.AddOption(&ser_ref_levels, "-rs", "--refine-serial",
+                   "Number of times to refine the mesh uniformly in serial.");
+    args.AddOption(&par_ref_levels, "-rp", "--refine-parallel",
+                   "Number of times to refine the mesh uniformly in parallel.");
+    args.AddOption(&order, "-o", "--order",
+                   "Order (degree) of the finite elements.");
+    args.AddOption(&ode_solver_type, "-s", "--ode-solver",
+                   "ODE solver: 1 - Backward Euler, 2 - SDIRK2, 3 - SDIRK3,\n\t"
+                   "\t   11 - Forward Euler, 12 - RK2, 13 - RK3 SSP, 14 - RK4.");
+    args.AddOption(&t_final, "-tf", "--t-final",
+                   "Final time; start time is 0.");
+    args.AddOption(&dt, "-dt", "--time-step",
+                   "Time step.");
+    args.AddOption(&alpha, "-a", "--alpha",
+                   "Alpha coefficient.");
+    args.AddOption(&kappa, "-k", "--kappa",
+                   "Kappa coefficient offset.");
+    args.AddOption(&radius, "-r", "--radius",
+                   "Radius of the radial source location.");
+    args.AddOption(&cx, "-cx", "--center_x",
+                   "Center offset in the x direction.");
+    args.AddOption(&cy, "-cy", "--center_y",
+                   "Center offset in the y direction.");
+    args.AddOption(&t_end, "-te", "--t_end",
+                   "End time of the source.");
+    args.AddOption(&amp, "-amp", "--amplitude",
+                   "Amplitude of the source.");
+    args.AddOption(&closest_rbf_val, "-crv", "--crv",
+                   "DMD Closest RBF Value.");
+    args.AddOption(&visualization, "-vis", "--visualization", "-no-vis",
+                   "--no-visualization",
+                   "Enable or disable GLVis visualization.");
+    args.AddOption(&visit, "-visit", "--visit-datafiles", "-no-visit",
+                   "--no-visit-datafiles",
+                   "Save data files for VisIt (visit.llnl.gov) visualization.");
+    args.AddOption(&vis_steps, "-vs", "--visualization-steps",
+                   "Visualize every n-th timestep.");
+    args.AddOption(&rdim, "-rdim", "--rdim",
+                   "Reduced dimension for DMD.");
+    args.AddOption(&offline, "-offline", "--offline", "-no-offline", "--no-offline",
+                   "Enable or disable the offline phase.");
```

Comment:
It might be helpful to define offline vs online phase.  If I run online phase, it looks like it solves the ODE, but then loads snapshots later on?  I'm thinking online phase shouldn't solve the ODE.